ambuda/models/base.py

```diff
@@ -20,6 +20,8 @@ def pk():
     return Column(Integer, primary_key=True, autoincrement=True)
 
 
-def foreign_key(field: str):
+def foreign_key(field: str, primary_key: bool = False):
```

Comment:
I think the `primary_key` option will be rare. Better to keep this function simple and inline the function body where you need it.